blog/2021-12-29-auto-setup.md

```diff
@@ -0,0 +1,121 @@
+---
+tags:
+  - Temporal
+  - Server
+posted_on_: 2021-12-29T00:00:09Z
+slug: auto-setup
+title: How Temporal Auto-Setup Works
+author: swyx
+author_title: Head of Developer Experience
+author_image_url: https://avatars.githubusercontent.com/u/6764957?v=4
+release_version: V1.14.0
+---
+
+We often say that “Temporal Server is a simple Go binary”, and yet most users are insulated from using this binary directly. There are several layers of abstraction to get through (docker-compose or Helm Chart, Dockerfile, and the auto-setup.sh script) before you reach this baseline understanding of Temporal. Together they comprise the environment requirements of Temporal:
```

Comment:
But is this really even true. Things like Temporalite prove you absolutely do not need Docker, Helm or others to have a "baseline understanding" of Temporal.

Reply:
i think if you read it again i'm saying the exact same thing - the baseline is what's underneath the docker, helm, other stuff